src/core/ddsc/include/dds/ddsc/dds_internal_api.h

```diff
@@ -43,12 +43,12 @@ extern "C" {
  * available samples issue #74. As a work around to support LENGTH_UNLIMITED in C++.
  * dds_reader_lock_samples() and @ref DDS_READ_WITHOUT_LOCK are needed.
  *
- * @param[in]   reader  Reader to lock the cache of.
+ * @param[in]   entity  Reader to lock the cache of.
  *
  * @returns the number of samples in the reader cache.
  */
 DDS_EXPORT uint32_t
-dds_reader_lock_samples (dds_entity_t reader);
```

Comment:
This really only takes a reader, so I think "reader" is the better choice. But see #1744. It is *finally* on its way out, so I don't care about this one! 🥳

Reply:
I care! (That's why fixed)